steps/s2i_steps.py

```diff
@@ -52,14 +52,16 @@ def s2i_inner(context, application, path='.', env="", incremental=False, tag="ma
 @given(u's2i build {application} from {path} with env using {tag}')
 @given(u's2i build {application} from {path} with env and {incremental}')
 @given(u's2i build {application} from {path} with env and {incremental} using {tag}')
-def s2i_build(context, application, path='.', env="", incremental=False, tag="master"):
+@given(u's2i build {application} from {path} with env and {incremental} using {tag} and run {run}')
+def s2i_build(context, application, path='.', env="", incremental=False, tag="master", run=True):
```

Comment:
Wouldn't it better to create a second method that would call this one. Then you could do:

```
@given(u's2i build {application} from {path} with env and {incremental} using {tag} without running'
```
on that method and call it with `run=False`. It seems for me more readable than this. Or just try to reword the `{run}` parameter.